SPECS/ncurses.spec

```diff
@@ -287,6 +287,9 @@ xz NEWS
 %{_libdir}/lib*.a
 
 %changelog
+* Mon Apr 28 2025 Yann Dirson <yann.dirson@valtes.tech> - 6.4-6.20240309
```

Comment:
```suggestion
* Mon Apr 28 2025 Yann Dirson <yann.dirson@vates.tech> - 6.4-6.20240309
```

Reply:
fixed... together with the old entries that caused this one to be mis-completed